html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1>Coding form!</h1>
+  <form>
+    <div class="email">
+      <label for="email-address">Email</label>
+      <div class="email-input">
+        <input type="text" id="email-address">
+      </div>
+    </div>
+    <label for="coding-choices">Why is coding cool?</label>
+    <div class="choices">
+      <select name="choices">
+        <option value="Beacause it's interesting">Beacause it's interesting</option>
+        <option value="Beacause it's fun">Beacause it's fun</option>
+        <option value="Beacause it's powerful">Beacause it's powerful</option>
+      </select>
+    </div>
+    <label for="langauges">Favourite language (choose one)</label>
+    <div class="radio-choices">
+      <input type="radio" name="language" value="html" check="checked" /> html
+      <input type="radio" name="language" value="css" check="checked" /> css
+      <input type="radio" name="language" value="Javascript" check="checked" /> Javascript
```

Comment:
The text for each option should be in a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!